skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -8,14 +8,36 @@ object SkikoMouseButtons {
     const val MIDDLE = 1 shl 2
 }
 
+enum class SkikoGestureEventKind {
+    PRESS, TAP, PAN, PINCH, ROTATION, LONGPRESS, SWIPE, UNKNOWN
+}
+enum class SkikoGestureEventDirection {
+    UP, DOWN, LEFT, RIGHT, UNKNOWN
+}
+enum class SkikoGestureEventState {
+    PRESS, STARTED, CHANGED, ENDED, UNKNOWN
+}
+expect class SkikoGesturePlatformEvent
+data class SkikoGestureEvent(
```

Comment:
Keep in mind, that Compose won't use it, as it read raw touch events, and convert them to gestures on Compose level.

But SkikoGestureEvent won't harm either, as it has nothing platform-specific, and can be commonized.